pages/data/stations/_id.vue

```diff
@@ -213,9 +283,136 @@ export default {
     this.$store.dispatch('stations/downloadStations')
       .then(() => {
         this.populate()
+        // this.refreshMetrics()
```

Comment:
The `this.$store.dispatch('stations/downloadStationsByDataset')` call actually makes the `stations/downloadStations` call too.  So we can move `this.populate()` over to the `stations/downloadStationsByDataset` success handler.